packages/applicant/src/pages/dashboard/index.page.tsx

```diff
@@ -52,13 +52,15 @@ export const getServerSideProps = async ({
   const oneLoginMatchingAccountBannerEnabled =
     process.env.ONE_LOGIN_MIGRATION_JOURNEY_ENABLED === 'true';
 
+  const oneLoginEnabled = process.env.ONE_LOGIN_ENABLED === 'true';
+
   return {
     props: {
       descriptionList,
       hasApplications,
-      oneLoginMatchingAccountBannerEnabled: Boolean(
-        oneLoginMatchingAccountBannerEnabled
-      ),
+      oneLoginMatchingAccountBannerEnabled:
+        oneLoginMatchingAccountBannerEnabled,
+      oneLoginEnabled: oneLoginEnabled,
     },
```

Comment:
Ultra minor point: we can shorthand these

```suggestion
      oneLoginMatchingAccountBannerEnabled,
      oneLoginEnabled
    },
```